extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -462,6 +464,27 @@ protected File getResourceAsTemporaryFile(final String resource) throws IOExcept
     return file;
   }
 
+  /**
+   * Helper method that populates a temporary file with {@code numRows} rows and {@code numColumns} columns where the
+   * first column is a string 'timestamp' while the rest are string columns with junk value
+   */
+  protected File generateTemporaryNdJsonFile(final int numRows, final int numColumns) throws IOException
```

Comment:
I think we can add a new class for this and the previous method. MSQTestFileUtils?